woocommerce/woocommerce
# Fix: Show preview label only when Product Collection block is selected

### Changes proposed in this Pull Request:





Updated the `ProductCollectionContent` component to display the preview label only when the Product Collection block itself is selected. Previously, the preview label was shown when either the block or any of its inner blocks were selected.

Key Changes:
- Removed the `isSelectedOrInnerBlockSelected` constant and its logic.
- Updated the condition to use `props.isSelected` directly to determine if the block is selected.

Closes #47645 



### How to test the changes in this Pull Request:



1. Go to "Products by Attribute" template
2. Add a Product Collection block if not already present
3. Verify that:
  - The preview label is displayed when the Product Collection block itself is selected (see screenshot below)
  - The preview label is not displayed when any of the inner blocks are selected
<img width="880" alt="image" src="https://github.com/woocommerce/woocommerce/assets/16707866/d34c6bea-8521-4ee7-8f94-17e3ec013be7">



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Fix: Show preview label only when Product Collection block is selected

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
